core/js/src/main/scala/zio/ZIOPlatformSpecific.scala

```diff
@@ -69,4 +69,17 @@ private[zio] trait ZIOCompanionPlatformSpecific { self: ZIO.type =>
       }
       promise.`then`[Unit](onFulfilled, js.defined(onRejected))
     }
+
+  def writeFile(file: => String, content: => String)(implicit trace: Trace): ZIO[Any, Throwable, Unit] = {
+    import scalajs.js.Dynamic.{global => g}
+    val fs = g.require("fs")
+    ZIO.attemptBlocking(fs.writeFileSync(file, content)).unit
```

Comment:
No need for `unit` here. Since you're only using it in one place I think you can just do `val fs = scalajs.js.Dynamic.global.requires("fs")` instead of importing it.